server/migrations/versions/3edc260ab0b1_audittype_batch_comparison.py

```diff
@@ -0,0 +1,32 @@
+# pylint: disable=invalid-name
+"""AuditType.BATCH_COMPARISON
+
+Revision ID: 3edc260ab0b1
+Revises: aa3978570981
+Create Date: 2020-09-28 22:11:46.110113+00:00
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "3edc260ab0b1"
+down_revision = "aa3978570981"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Alembic doesn't support changing enums, so we have to do with with raw
+    # SQL. You can't alter an enum inside a transaction block (which this
+    # function is wrapped in by Alembic), so first we have to commit the
+    # transaction.
+    # https://github.com/sqlalchemy/alembic/issues/123#issuecomment-441882887
+    op.execute("COMMIT")
+    op.execute("ALTER TYPE audittype ADD VALUE 'BALLOT_COMPARISON'")
```

Comment:
Yeah, this is one reason I haven't always enjoyed enums, but alright.

Is it worth doing an `op.execute("BEGIN TRANSACTION")` afterwards so the rest is under a new transaction?

Reply:
what do you mean "the rest"?

Reply:
maybe I misunderstood, is it one transaction around each upgrade file, or around all of them at once? I meant the rest of the upgrade files. If it's one transaction per upgrade file, then ignore my comment.

Reply:
ah i figured out what you mean. i didn't realize the entire set of migrations was wrapped in one transaction. given that, I agree that this is not a great approach. found a better one: bb48e0d

Reply:
much nicer!